aibolit/metrics/ncss/ncss.py

```diff
@@ -32,19 +34,16 @@ def value(self, filename: str):
         if len(filename) == 0:
             raise ValueError('Empty file for analysis')
 
-        tree = build_ast(filename)
-
+        tree = AST.build_from_javalang(build_ast(filename))
         metric = 0
-        for _, node in tree:
-            node_type = str(type(node))
-            if 'Statement' in node_type:
-                metric += 1
-            elif 'VariableDeclarator' == node_type:
-                metric += 1
-            elif 'Assignment' == node_type:
-                metric += 1
-            elif 'Declaration' in node_type and 'LocalVariableDeclaration' not in node_type \
-                 and 'PackageDeclaration' not in node_type:
-                metric += 1
+        for _, destination, edge_type in dfs_labeled_edges(tree.tree, tree.root):
+            if edge_type == 'forward':
+                node_type = str(tree.get_type(destination)).split('.')[1]
+                if 'STATEMENT' in node_type and 'BLOCK_STATEMENT' not in node_type:
+                    metric += 1
+                elif node_type == 'CATCH_CLAUSE':
```

Comment:
We better compare enum items, than strings.

Reply:
ok